stubs/psutil/psutil/_common.pyi

```diff
@@ -14,20 +15,39 @@ NETBSD: bool
 BSD: bool
 SUNOS: bool
 AIX: bool
-STATUS_RUNNING: str
-STATUS_SLEEPING: str
-STATUS_DISK_SLEEP: str
-STATUS_STOPPED: str
-STATUS_TRACING_STOP: str
-STATUS_ZOMBIE: str
-STATUS_DEAD: str
-STATUS_WAKE_KILL: str
-STATUS_WAKING: str
-STATUS_IDLE: str
-STATUS_LOCKED: str
-STATUS_WAITING: str
-STATUS_SUSPENDED: str
-STATUS_PARKED: str
+
+STATUS_RUNNING: Literal["running"]
+STATUS_SLEEPING: Literal["sleeping"]
+STATUS_DISK_SLEEP: Literal["disk-sleep"]
+STATUS_STOPPED: Literal["stopped"]
+STATUS_TRACING_STOP: Literal["tracing-stop"]
+STATUS_ZOMBIE: Literal["zombie"]
+STATUS_DEAD: Literal["dead"]
+STATUS_WAKE_KILL: Literal["wake-kill"]
+STATUS_WAKING: Literal["waking"]
+STATUS_IDLE: Literal["idle"]
+STATUS_LOCKED: Literal["locked"]
+STATUS_WAITING: Literal["waiting"]
+STATUS_SUSPENDED: Literal["suspended"]
+STATUS_PARKED: Literal["parked"]
+
+_Status = Literal[
```

Comment:
It would be much cooler if `_Status = Literal[STATUS_RUNNING, STATUS_SLEEPING, ...]` was possible 😞